dbms/src/Storages/Page/V3/PageDirectory.h

```diff
@@ -282,7 +287,14 @@ class VersionedPageEntries
         , create_ver(0)
         , delete_ver(0)
         , ori_page_id{}
-    {}
+    {
+        PageStorageMemorySummary::mem_sum_page_entries.fetch_add(sizeof(VersionedPageEntries<Trait>));
```

Comment:
VersionedPageEntries is mutable. Maybe better to update the memory count when it is modified (like `createDelete`)?

Reply:
We don't track VersionedPageEntries